internal/pkg/client/cache/dir.go

```diff
@@ -42,17 +42,6 @@ func Root() string {
 	return root
 }
 
-// Clean : wipes all files in the cache directory, will return a error if one occurs
```

Comment:
By removing this public function, it breaks at least the unit tests. I do not believe that the goal at the moment is to revisit the API, especially the public API, but rather implement a "quick fix". Changing the public API would also impact all the other efforts to correctly image caches and it is unclear to me how that would be beneficial from a timing point of view and coordination of efforts.